src/openforms/registrations/contrib/objects_api/templatetags/registrations/contrib/objects_api/json_tags.py

```diff
@@ -32,3 +32,14 @@ def json_summary(context):
         json_data = html_escape_json(json_data)
 
     return SafeString(json.dumps(json_data))
+
+
+@register.simple_tag
+def as_geo_json(value: list[float]) -> SafeString:
+    """Output the ``value`` as a safe GeoJSON dumped string.
+
+    As of today, this only supports coordinates.
```

Comment:
Please cross reference the `_transform_coordinates` helper - they go hand in hand.